qrisk/stats.py

```diff
@@ -62,33 +64,35 @@ def cum_returns(returns, starting_value=None):
     -----
     For increased numerical accuracy, convert input to log returns
     where it is possible to sum instead of multiplying.
+    PI((1+r_i)) - 1 = exp(ln(PI(1+r_i)))     # x = exp(ln(x))
+                    = exp(SIGMA(ln(1_r_i))   # ln(a*b) = ln(a) + ln(b)
     """
 
     # df_price.pct_change() adds a nan in first position, we can use
-    # that to have cum_returns start at the origin so that
+    # that to have cum_logarithmic_returns start at the origin so that
     # df_cum.iloc[0] == starting_value
     # Note that we can't add that ourselves as we don't know which dt
     # to use.
     if pd.isnull(returns.iloc[0]):
         returns.iloc[0] = 0.
 
-    df_cum = np.exp(np.log(1 + returns).cumsum())
+    df_cum = np.exp(np.log1p(returns).cumsum())
 
-    if starting_value is None:
+    if starting_value == 0:
         return df_cum - 1
     else:
         return df_cum * starting_value
 
 
-def aggregate_returns(df_daily_rets, convert_to):
+def aggregate_returns(returns, convert_to):
     """
     Aggregates returns by week, month, or year.
 
     Parameters
     ----------
-    df_daily_rets : pd.Series
+    returns : pd.Series
        Daily returns of the strategy, noncumulative.
-        - See full explanation in tears.create_full_tear_sheet (returns).
+        - See full explanation in :func:`cum_returns`.
```

Comment:
I'm not very familiar with sphinx, would that be :func:`~qrisk.stats.cum_function` or :func:`~stats.cum_function`?